sematic/ui/src/pipelines/PipelineBar.tsx

```diff
@@ -270,6 +270,9 @@ export default function PipelineBar() {
                         <code>{id.substring(0, 6)}</code>
                       </Typography>
                     </Box>
+                    <Box sx={{marginLeft: '10px'}}>
```

Comment:
We should use the standard MUI spacing units no?
```typescript
<Box sx={{ml: 3}}>
```
like below.
Or use a grid display with spacing of `3` between elements.

Reply:
Thanks! Fixed.